concourse/steps/component_descriptor.mako

```diff
@@ -45,6 +45,7 @@ with open(version_file_path) as f:
   effective_version = f.read().strip()
 
 component_name = '${descriptor_trait.component_name()}'
+component_labels = ${descriptor_trait.component_descriptor_labels()}
```

Comment:
I think I already asked that: will this work if the attr is not specified (as in: will the result be valid yaml)? caveat (sorry for "tricking" you into this: I think, the tuple literal is not compatible w/ yaml, is it?

Reply:
Yes, pyyaml will dump tuples as lists, but when using `safe_dump` the type-information is lost. On deserialisation the labels will be in a list.

Reply:
I just tried:
```>>> yaml.dump({'a': ()})
'a: !!python/tuple []\n'
```

so I figure pyyaml will retain the tuple-metadata. As the component-descriptor-spec requires a _list_, I think we should also actually dump an (empty) list